pallets/avn/src/tests/mock.rs

```diff
@@ -86,61 +158,129 @@ parameter_types! {
 
 thread_local! {
     // validator accounts (aka public addresses, public keys-ish)
-    pub static VALIDATORS: RefCell<Option<Vec<u64>>> = RefCell::new(Some(vec![1, 2, 3]));
+    pub static VALIDATOR_SEEDS: RefCell<Option<Vec<u64>>> = RefCell::new(Some(vec![1, 2, 3]));
 }
 
 pub type SessionIndex = u32;
+pub type ValidatorId = <Test as session::Config>::ValidatorId;
 
-pub struct TestSessionManager;
-impl session::SessionManager<u64> for TestSessionManager {
-    fn new_session(_new_index: SessionIndex) -> Option<Vec<u64>> {
-        VALIDATORS.with(|l| l.borrow_mut().take())
-    }
-    fn end_session(_: SessionIndex) {}
-    fn start_session(_: SessionIndex) {}
-}
-
-impl session::Config for TestRuntime {
-    type SessionManager = TestSessionManager;
+impl session::Config for Test {
```

Comment:
You should run `cargo +nightly fmt` to format the code, otherwise the CI will complain.